app/src/main/res/values-de/strings.xml

```diff
@@ -226,6 +229,10 @@
     <string name="mark_notification_summary">Benachrichtigung als gelesen markieren sobald sie angetippt wurde.</string>
     <string name="theme_title">Wähle ein Design</string>
     <string name="theme_summary">Wähle das Standarddesign aus</string>
+	<string name="theme_color_summary">Wähle eine Akzentfarbe</string>
+    <string name="theme_color_title">Akzentfarbe</string>
+    <string name="logo_designer" formatted="true" translatable="false">Logo Designer (%s)</string>
+    <string name="google_plus" translatable="false">Google+</string>
```

Comment:
"cat app/src/main/res/values-de/strings.xml | grep 'translatable="false"' | wc -l" gives 5 back so I need to remove them all? I don't think that they'll conflict since the strings.xml file from some other translators also contains these lines and some of them even translated them so what exactly is wrong?